efficientdet/keras/train.py

```diff
@@ -140,13 +139,13 @@ def main(_):
       strategy=FLAGS.strategy,
       batch_size=FLAGS.batch_size,
       num_shards=ds_strategy.num_replicas_in_sync)
-
+  config.__dict__.update(params)
```

Comment:
config.override(params)

or 
config.override(params, allow_new_keys=True) if you want allow new members